code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1197,7 +1197,10 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 		var/drink_size = to_eat.amount_per_transfer_from_this > 5 ? 5 : to_eat.amount_per_transfer_from_this
 		if(fraction)
 			to_eat.reagents.reaction(src, REAGENT_INGEST, fraction)
-			to_eat.reagents.trans_to(src, drink_size)
+			if(drinksize_override)
+				to_eat.reagents.trans_to(src, drinksize_override)
+			else
+				to_eat.reagents.trans_to(src, drink_size)
```

Comment:
that won't work. I could however set a default value for argument:
`drinksize_ovveride = to_eat.amount_per_transfer_from_this > 5 ? 5 : to_eat.amount_per_transfer_from_this`
and use that instead. As matter of fact might as well just call it drink_size. This will however make function declaration less readable
That way there will also be no need in declaring var/drink_size, as it will take default instead